<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="IntegerButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Integer Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DoubleButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Double Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StringButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>String Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <zorder>BackButton</zorder>
   <zorder>IntegerButton</zorder>
   <zorder>DoubleButton</zorder>
   <zorder>StringButton</zorder>
   <zorder>ExitButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
